doc/apiv3-documentation.apib

```diff
@@ -1991,9 +2003,9 @@ This endpoint lists the types that are *available* in a given project.
                 "lastName": "Sheppard",
                 "mail": "shep@mail.com",
                 "avatar": "https://gravatar/avatar",
+				"status": "active",
```

Comment:
Thanks for that one... I managed to introduce >1000 tabs with this PR ^^

Editor settings are now fixed again.